addons/purchase/__manifest__.py

```diff
@@ -38,6 +38,7 @@
     'application': True,
     'assets': {
         'web.assets_backend': [
+            'purchase/static/src/toaster_button/*',
```

Comment:
@aab-odoo I don't remember if we can do that ? Are the xml ignored properly ?

Reply:
yes